contracts/test/ENSTest.sol

```diff
@@ -103,14 +127,86 @@ contract TestENSRegistry is ENSRegistry {
         emit NewTTL(_node, _ttl);
         records[_node].ttl = _ttl;
     }
+
+    function _setResolverAndTTL(bytes32 _node, address _resolver, uint64 _ttl) internal {
+        if(_resolver != records[_node].resolver) {
+            records[_node].resolver = _resolver;
+            emit NewResolver(_node, _resolver);
+        }
+
+        if(_ttl != records[_node].ttl) {
+            records[_node].ttl = _ttl;
+            emit NewTTL(_node, _ttl);
+        }
+    }
+}
+
+contract ENSRegistryWithFallback is ENSRegistry {
```

Comment:
I believe it should extend `TestENSRegistry` as `ENSRegistry is just an interface.